crates/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -405,6 +405,7 @@ fn execute_move_publish<E: fmt::Debug, S: StorageView<E>, Mode: ExecutionMode>(
         })
         .collect::<Vec<_>>();
 
+    // new_package also initializes type origin table in the package object
```

Comment:
Heads up, you may need to rebase -- I believe @tzakian has landed his changes to introduce `Upgrade` scaffolding, and that will affect this function.

Reply:
Also a heads up for #9135 which will make the dependency information available to you in the command.